File: sdk/client-ts/src/___PLAYGROUND___/authentication/test-auth.ts
import { begin } from '../helpers';

export const testAuth = async () => {
  const context = await begin();

  console.log('Testing authentication...');

  // Test auth by getting networks (requires valid auth)
  try {
    const networks = await context.client.network.getAll();
    console.log('Authentication successful - got networks:', networks.length);
  } catch (error) {
    console.error('Authentication failed:', error);
  }

  await context.end();
};

testAuth().then(() => console.log('test-auth done'));

File: sdk/python/src/__PLAYGROUND__/authentication/test-auth.py
import asyncio

from __PLAYGROUND__.helper import begin, end


async def testAuth():
    try:
        client, _, _, rrelayer_node = await begin()

        print("Testing authentication...")
        networks = await client.network.getAll()
        print("Authentication successful - got networks:", networks)
    except Exception as e:
        print("Authentication failed:", e)
    finally:
        end(rrelayer_node)


if __name__ == "__main__":
    asyncio.run(testAuth())
    print("test-auth done")
